src/main/java/pl/javastart/task/Grade.java

```diff
@@ -0,0 +1,32 @@
+package pl.javastart.task;
+
+public class Grade {
+
+    private int studentIndex;
+    private String groupId;
```

Comment:
Tutaj też wygodniej byłoby przechowywać referencje, a nie same identyfikatory, wtedy przy wyświetlaniu kod się uprości nie będzie trzeba robić tylu wyszukiwań w pętlach

Reply:
myślałem o tym, ale miałem zaćmienie i nie wiedziałem jak to zrobić